docs/src/Atmos/SurfaceFluxes.md

```diff
@@ -0,0 +1,114 @@
+# `SurfaceFluxes`
+
+```@meta
+CurrentModule = CLIMA.SurfaceFluxes
+```
+
+Surface flux functions, e.g. for buoyancy flux, friction velocity, and exchange coefficients.
+
+## `Byun1990`
+
+Compute surface fluxes using the approach in Byun (1990).
+
+### Plots
+
+```@example byun1990
+using CLIMA.SurfaceFluxes.Byun1990
+using Plots, LaTeXStrings
+
+Ri_range = range(-1.2, stop=0.24, length=100)
+scales = [50,200,600,1000,10_000]
+
+z_0 = 1.0
+γ_m, γ_h = 15.0, 9.0
+β_m, β_h = 4.8, 7.8
+Pr_0 = 0.74
+
+plot(Ri_range,
+    [Byun1990.compute_exchange_coefficients(Ri,scale*z_0,z_0,γ_m,γ_h,β_m,β_h,Pr_0)[1]
+        for Ri in Ri_range, scale in scales], 
+    xlabel="Bulk Richardson number (Ri_b)", ylabel="Drag coefficient", title="Momentum exchange coefficient",
+    labels=scales, legendtitle=L"z/z_0")
+
+savefig("exchange_byun1990_fig4a.svg") # hide
+nothing # hide
+```
+![](exchange_byun1990_fig4a.svg)
+
+Recreation of Figure 4(a) from Byun (1990)
+
+```@example byun1990
+plot(Ri_range,
+    [Byun1990.compute_exchange_coefficients(Ri,scale*z_0,z_0,γ_m,γ_h,β_m,β_h,Pr_0)[2]
+        for Ri in Ri_range, scale in scales], 
+    xlabel="Bulk Richardson number (Ri_b)", ylabel="Exchange coefficient", title="Heat exchange coefficient",
+    labels=scales, legendtitle=L"z/z_0")
+
+savefig("exchange_byun1990_fig4b.svg") # hide
+nothing # hide
+```
+![](exchange_byun1990_fig4b.svg)
+
+Recreation of Figure 4(b) from Byun (1990)
+
+## `Nishizawa2018`
+
+### Plots
+```@example
+using CLIMA.SurfaceFluxes.Nishizawa2018
+using Plots, LaTeXStrings
+
+L, a, θ, z, flux = rand(5,1)
+z = z/100
+
+u = range(-0.1, stop=0.1, length=100)
+L = Nishizawa2018.compute_MO_len.(u, θ, flux)
+ζ = z./L
+ϕ_m = Nishizawa2018.compute_ϕ_m.(ζ, L, a)
```

Comment:
this function isn't defined. I guess it got renamed somewhere?

Reply:
We inlined it in #64, we can revive these functions if we want to use them to recreate the plots. I'm fine either way. They're relatively small functions, so I don't know that we need to plot them for verification.